### DISCUSSION TITLE
Cloned drive created the same ID on both PCs, and am unable to change it

### DISCUSSION THREAD

**Discussion by 4C3T** [rustdesk/rustdesk#5858] [Q&A]
Hi. I needed to clone my PCIe NVMe SSD from another PC which uses Clover UEFI bootmanager to boot, and this caused the problem of Rustdesk getting the same ID for both computers, and when I try to change it, it won't allow it. It can't connect to a server apparently, and I feel that this should be possible to do locally.
_The reason for cloning was that I'm unable to install on the PC I'm running it from. I can run Clover UEFI just fine on it, but Windows can't install from/via it, so the simple solution was to just copy an existing drive/partition._

Tried a while back to find the string in Windows registry, but came up with nothing (Didn't search thoroughly though as searches tend to lag and I was running stuff that didn't need interruptions at the time so I did one search for the ID but found nothing).

Also tried removing it and re-installing it with no success (Hence the registry-attempt), so I don't know what to do next.

**Comment by GPOSCLIC** [comment]
I tried the following (with success) on Win10 PC (VirtualBox):
- terminated RustDesk
- deleted RustDesk folders in %APPDATA% and %LOCALAPPDATA% directory.
- switched the MAC address of NIC to a random address, using 'macshift' util.
- starting rustdesk.exe - it has new ID but cannot connect.
- switched the MAC address of NIC to original address, using 'macshift' util.
- starting rustdesk.exe - it has new ID and connects.
best regards.
